docs/OnlineTutorial/guide.md

```diff
@@ -852,9 +852,9 @@ can do something like implement the mathematical Fibonacci function:
 ``` {.edit}
 function fib(n: nat): nat
 {
-   if n == 0 then 0 else
-   if n == 1 then 1 else
-                  fib(n - 1) + fib(n - 2)
+  if n == 0 then 0 else
+  if n == 1 then 1 else
+                 fib(n - 1) + fib(n - 2)
```

Comment:
```suggestion
  if n == 0 then 0 else
  if n == 1 then 1 else
  fib(n - 1) + fib(n - 2)
```

Reply:
As before -- I think the second line deserves some indentation